docs/upgrade_history/v27_evm_emulation/v27-evm-emulation.md

```diff
@@ -0,0 +1,56 @@
+# V27 EVM emulation upgrade
+
+## Upgrade process
+
+V27 upgrade will happen after the gateway preperation upgrade, but before the gateway is deployed. As such the ugprade process does not involve the Gateway parts ( upgrading the CTM on GW, pausing migrations, etc), it is an L1-only upgrade. 
+
+Additionally this is not a bridge upgrade, as the bridge and ecosystem contracts don't have new features, so L1<>L2 bridging is not affected. This means that only the system components, the Verifiers, Facets and L2 contracts need to be upgraded.
+
+The upgrade process is as follows: 
+
+- deploy new contract implementations. 
+- publish L2 bytecodes
+- generate upgrade data
+    - forceDeployments data on L2. Contains all new System and L2 contracts.
+    - new genesis diamondCut (which contains facetCuts, and genesis forceDeployments, as well as init data)
+    - upgradeCut (which contains facetCuts, and upgrade forceDeployments, as well as upgrade data)
+- prepare ecosytem upgrade governance calls:
+    - upgrade proxies for L1 contracts. 
+    - CTM: 
+        - set new ChainCreationParams (-> contains new genesis upgrade cut data)
+        - set new version upgrade (-> contains new upgrade cut data)
+    - start upgrade timer
+
+Read more here: [Upgrade process document](../../chain_management/upgrade_process.md)
+
+## Changes: 
+
+### New features: 
+
+- facets: EVM emulation, service transaction on Mailbox
```

Comment:
I would move EVM emulation to a separate item here

Reply:
I had it in the next line as well